diff --git a/fallCollection.twig b/fallCollection.twig
new file mode 100644
index 0000000..4a6ec60
--- /dev/null
+++ b/fallCollection.twig
@@ -0,0 +1,3 @@
+<h1>
+    <!-- print the variable here -->
+</h1>
diff --git a/render.php b/render.php
index fb72eaa..e55df8b 100644
--- a/render.php
+++ b/render.php
@@ -11,3 +11,9 @@ $loader = new \Twig_Loader_Filesystem([
 $twig = new \Twig_Environment($loader, [
     'debug' => true
 ]);
+
+$output = $twig->render('fallCollection.twig', [
+    'collectionTitle' => 'Fall in love and look your best in the snow.'
+]);
+
+echo $output;
