diff --git a/fallCollection.twig b/fallCollection.twig
new file mode 100644
index 0000000..4a6ec60
--- /dev/null
+++ b/fallCollection.twig
@@ -0,0 +1,3 @@
+<h1>
+    <!-- print the variable here -->
+</h1>
diff --git a/render.php b/render.php
index 8e39ce9..1ef42ca 100644
--- a/render.php
+++ b/render.php
@@ -14,7 +14,9 @@ $twig = new \Twig_Environment($loader, [
 ]);
 
 try {
-    $output = $twig->render('fallCollection.twig');
+    $output = $twig->render('fallCollection.twig', [
+        'collectionTitle' => 'Fall in love and look your best in the snow.'
+    ]);
 } catch (\Twig_Error_Syntax $e) {
     // this tells the grader that there was an error
     $_ENV['language_error'] = true;
