diff --git a/layout.twig b/layout.twig
index f53d311..685b686 100644
--- a/layout.twig
+++ b/layout.twig
@@ -15,7 +15,16 @@
         </div>
     </nav>
 
-    {% block body %}{% endblock %}
+    <div class="container">
+        <div class="row">
+            <div class="col-xs-3">
+                {% block sidebar %}{% endblock %}
+            </div>
+            <div class="col-xs-9">
+                {% block body %}{% endblock %}
+            </div>
+        </div>
+    </div>
 
     <footer class="footer">
         {% block footer %}
