diff --git a/fallCollection.twig b/fallCollection.twig
index 53bb3f4..48c181f 100644
--- a/fallCollection.twig
+++ b/fallCollection.twig
@@ -16,12 +16,12 @@
             {{ product.name }}
             <span class="price">{{ product.price }}</span>
         </li>
+    {% else %}
+        <li>
+            <div class="alert alert-warning">
+                No pants for you!
+            </div>
+        </li>
     {% endfor %}
     </ul>
-
-    {% if products is empty %}
-        <div class="alert alert-warning">
-            No pants for you!
-        </div>
-    {% endif %}
 {% endblock %}
