diff --git a/fallCollection.twig b/fallCollection.twig
index fcc3b44..d3f8adf 100644
--- a/fallCollection.twig
+++ b/fallCollection.twig
@@ -1,19 +1,11 @@
 {% extends 'layout.twig' %}
 
 {% block body %}
-    {{ include('_featuredProduct.twig') }}
-
     <h1>
         {{ collectionTitle }}
         <small>{{ fallCollectionPromoDescription }}</small>
     </h1>
 
-    {% if products|length is odd %}
-        <div class="alert alert-info">
-            We promise, 1 more pair of pants is coming very soon!
-        </div>
-    {% endif %}
-
     <ul class="list-group">
     {% for product in products %}
         <li class="list-group-item">
@@ -21,18 +13,15 @@
                 {{ random(['black', 'white', 'green']|upper) }}
             </span>
 
-            <!-- fix this to print the product's name -->
             {{ product.name }}
-
-            <span class="price">
-                {{ product.price }}
-            </span>
-            (release date: {{ product.futureReleaseDate|date('Y-m-d') }})
+            <span class="price">{{ product.price }}</span>
         </li>
     {% endfor %}
     </ul>
 {% endblock %}
 
-{% block footer %}
-    Winter is coming! Get your pants!
-{% endblock %}
+{% if products is empty %}
+    <div class="alert alert-warning">
+        No pants for you!
+    </div>
+{% endif %}
diff --git a/render.php b/render.php
index 8df8b40..318c984 100644
--- a/render.php
+++ b/render.php
@@ -22,6 +22,7 @@ $products = array(
     new \PantsProduct('South Shore Swim Shorts', 49),
     new \PantsProduct('Starfish Halloween Costume', 35)
 );
+$products = [];
 
 try {
     $output = $twig->render('fallCollection.twig', [
