diff --git a/fallCollection.twig b/fallCollection.twig
index d11ffd1..72598d4 100644
--- a/fallCollection.twig
+++ b/fallCollection.twig
@@ -1,7 +1,21 @@
+<!DOCTYPE html><html lang="en"><head><link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"><link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css"></head>
+<body>
+
 <h1>
     {{ collectionTitle }}
 </h1>
 
+<p class="lead">
+    The sale starts: <!-- print the date here -->
+</p>
+
+<ul class="list-group">
 {% for product in products %}
-	<h3>{{ product }}</h3>
+	<li class="list-group-item">
+        {{ product }}
+    </li>
 {% endfor %}
+</ul>
+
+</body>
+</html>
\ No newline at end of file
diff --git a/render.php b/render.php
index 7f5eeba..dbe9b5e 100644
--- a/render.php
+++ b/render.php
@@ -14,6 +14,8 @@ $twig = new \Twig_Environment($loader, [
 ]);
 $twig->addExtension(new \Twig_Extension_Debug());
 
+$saleStartsAt = new \DateTime('+1 week');
+
 try {
     $output = $twig->render('fallCollection.twig', [
         'collectionTitle' => 'Fall in love and look your best in the snow.',
@@ -22,7 +24,8 @@ try {
             'Antarctic Snow Pants (in leopard seal print)',
             'South Shore Swim Shorts',
             'Starfish Halloween Costume'
-        )
+        ),
+        'saleStartsAt' => $saleStartsAt
     ]);
 } catch (\Twig_Error_Syntax $e) {
     // this tells the grader that there was an error
