diff --git a/fallCollection.twig b/fallCollection.twig
index 3d7f4d5..9810d57 100644
--- a/fallCollection.twig
+++ b/fallCollection.twig
@@ -3,6 +3,9 @@
 
 <h1>
     {{ collectionTitle }}
+    <small>
+        <!-- print the description here -->
+    </small>
 </h1>
 
 <ul class="list-group">
diff --git a/render.php b/render.php
index dbe9b5e..0e43e75 100644
--- a/render.php
+++ b/render.php
@@ -15,6 +15,7 @@ $twig = new \Twig_Environment($loader, [
 $twig->addExtension(new \Twig_Extension_Debug());
 
 $saleStartsAt = new \DateTime('+1 week');
+$fallCollectionPromoDescription = 'Still wearing your summer swim shorts? Time to plan for the 9 month winter in style. Look no further than our fall collection at Penguins Pants Plus!';
 
 try {
     $output = $twig->render('fallCollection.twig', [
@@ -25,7 +26,8 @@ try {
             'South Shore Swim Shorts',
             'Starfish Halloween Costume'
         ),
-        'saleStartsAt' => $saleStartsAt
+        'saleStartsAt' => $saleStartsAt,
+        'fallCollectionPromoDescription' => $fallCollectionPromoDescription,
     ]);
 } catch (\Twig_Error_Syntax $e) {
     // this tells the grader that there was an error
