diff --git a/fallCollection.twig b/fallCollection.twig
index 97420e1..3d7f4d5 100644
--- a/fallCollection.twig
+++ b/fallCollection.twig
@@ -9,8 +9,7 @@
 {% for product in products %}
 	<li class="list-group-item">
         <span class="badge">
-            <!-- print black, white or green here randomly -->
-            <!-- and make them all capital letters -->
+            {{ random(['black', 'white', 'green'])|upper }}
         </span>
 
         {{ product }}
