diff --git a/fallCollection.twig b/fallCollection.twig
index b2ac915..97420e1 100644
--- a/fallCollection.twig
+++ b/fallCollection.twig
@@ -5,17 +5,18 @@
     {{ collectionTitle }}
 </h1>
 
-<p class="lead">
-    The sale starts: {{ saleStartsAt|date('F jS') }}
-</p>
-
 <ul class="list-group">
 {% for product in products %}
 	<li class="list-group-item">
+        <span class="badge">
+            <!-- print black, white or green here randomly -->
+            <!-- and make them all capital letters -->
+        </span>
+
         {{ product }}
     </li>
 {% endfor %}
 </ul>
 
 </body>
-</html>
\ No newline at end of file
+</html>
