diff --git a/layout.twig b/layout.twig
index 34d58ff..5f2da7f 100644
--- a/layout.twig
+++ b/layout.twig
@@ -3,6 +3,7 @@
 <head>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css">
+    <link rel="stylesheet" href="https://getbootstrap.com/examples/jumbotron-narrow/jumbotron-narrow.css">
 </head>
 <body>
 
@@ -16,5 +17,8 @@
 
     {% block body %}{% endblock %}
 
+    <footer class="footer">
+        You're hip, you're cool, you're a penguin!
+    </footer>
 </body>
 </html>
\ No newline at end of file
