diff --git a/fallCollection.twig b/fallCollection.twig
index 5693171..8e6e8c7 100644
--- a/fallCollection.twig
+++ b/fallCollection.twig
@@ -1,19 +1,6 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css">
-</head>
-<body>
-
-    <nav class="navbar navbar-inverse">
-        <div class="container">
-            <div class="navbar-header">
-                <a class="navbar-brand" href="#">Penguin Pants Plus</a>
-            </div>
-        </div>
-    </nav>
+{% extends 'layout.twig' %}
 
+{% block body %}
     <h1>
         {{ collectionTitle }}
         <small>{{ fallCollectionPromoDescription }}</small>
@@ -36,6 +23,4 @@
         </li>
     {% endfor %}
     </ul>
-
-</body>
-</html>
+{% endblock %}
diff --git a/layout.twig b/layout.twig
index e69de29..34d58ff 100644
--- a/layout.twig
+++ b/layout.twig
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css">
+</head>
+<body>
+
+    <nav class="navbar navbar-inverse">
+        <div class="container">
+            <div class="navbar-header">
+                <a class="navbar-brand" href="#">Penguin Pants Plus</a>
+            </div>
+        </div>
+    </nav>
+
+    {% block body %}{% endblock %}
+
+</body>
+</html>
\ No newline at end of file
diff --git a/mainCollection.twig b/mainCollection.twig
index 489c8f9..5178a8b 100644
--- a/mainCollection.twig
+++ b/mainCollection.twig
@@ -1,22 +1,7 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css">
-</head>
-<body>
-
-    <nav class="navbar navbar-inverse">
-        <div class="container">
-            <div class="navbar-header">
-                <a class="navbar-brand" href="#">Penguin Pants Plus</a>
-            </div>
-        </div>
-    </nav>
+{% extends 'layout.twig' %}
 
+{% block body %}
     <h1>Welcome to our main collection or products</h1>
 
     <div>The main products are being updated!</div>
-
-</body>
-</html>
\ No newline at end of file
+{% endblock %}
