diff --git a/fallCollection.twig b/fallCollection.twig
index 07a5b0a..5693171 100644
--- a/fallCollection.twig
+++ b/fallCollection.twig
@@ -1,28 +1,41 @@
-<!DOCTYPE html><html lang="en"><head><link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"><link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css"></head>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css">
+</head>
 <body>
 
-<h1>
-    {{ collectionTitle }}
-    <small>{{ fallCollectionPromoDescription }}</small>
-</h1>
+    <nav class="navbar navbar-inverse">
+        <div class="container">
+            <div class="navbar-header">
+                <a class="navbar-brand" href="#">Penguin Pants Plus</a>
+            </div>
+        </div>
+    </nav>
 
-<ul class="list-group">
-{% for product in products %}
-	<li class="list-group-item">
-        <span class="badge">
-            {{ random(['black', 'white', 'green'])|upper }}
-        </span>
+    <h1>
+        {{ collectionTitle }}
+        <small>{{ fallCollectionPromoDescription }}</small>
+    </h1>
 
-        <!-- fix this to print the product's name -->
-        {{ product.name }}
+    <ul class="list-group">
+    {% for product in products %}
+        <li class="list-group-item">
+            <span class="badge">
+                {{ random(['black', 'white', 'green'])|upper }}
+            </span>
 
-        <span class="price">
-            {{ product.price }}
-        </span>
-        (release date: {{ product.futureReleaseDate|date('Y-m-d') }})
-    </li>
-{% endfor %}
-</ul>
+            <!-- fix this to print the product's name -->
+            {{ product.name }}
+
+            <span class="price">
+                {{ product.price }}
+            </span>
+            (release date: {{ product.futureReleaseDate|date('Y-m-d') }})
+        </li>
+    {% endfor %}
+    </ul>
 
 </body>
 </html>
diff --git a/layout.twig b/layout.twig
new file mode 100644
index 0000000..e69de29
diff --git a/mainCollection.twig b/mainCollection.twig
new file mode 100644
index 0000000..489c8f9
--- /dev/null
+++ b/mainCollection.twig
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css">
+</head>
+<body>
+
+    <nav class="navbar navbar-inverse">
+        <div class="container">
+            <div class="navbar-header">
+                <a class="navbar-brand" href="#">Penguin Pants Plus</a>
+            </div>
+        </div>
+    </nav>
+
+    <h1>Welcome to our main collection or products</h1>
+
+    <div>The main products are being updated!</div>
+
+</body>
+</html>
\ No newline at end of file
