diff --git a/fallCollection.twig b/fallCollection.twig
index ba08ca3..dd90559 100644
--- a/fallCollection.twig
+++ b/fallCollection.twig
@@ -8,6 +8,11 @@
     </small>
 </h1>
 
+<h3>
+    <span class="glyphicon glyphicon-star"></span>
+    <!-- print a random featured product here! -->
+</h3>
+
 <ul class="list-group">
 {% for product in products %}
 	<li class="list-group-item">
diff --git a/render.php b/render.php
index 0e43e75..226640d 100644
--- a/render.php
+++ b/render.php
@@ -28,6 +28,7 @@ try {
         ),
         'saleStartsAt' => $saleStartsAt,
         'fallCollectionPromoDescription' => $fallCollectionPromoDescription,
+        'randomProductKey' => rand(0, 3)
     ]);
 } catch (\Twig_Error_Syntax $e) {
     // this tells the grader that there was an error
