diff --git a/fallCollection.twig b/fallCollection.twig
index 3dc5fb5..f799506 100644
--- a/fallCollection.twig
+++ b/fallCollection.twig
@@ -8,6 +8,8 @@
         <small>{{ fallCollectionPromoDescription }}</small>
     </h1>
 
+    {# Print your message here if there are an odd number of pants! #}
+
     <ul class="list-group">
     {% for product in products %}
         <li class="list-group-item">
diff --git a/render.php b/render.php
index f8862b4..4d9b5d8 100644
--- a/render.php
+++ b/render.php
@@ -21,7 +21,7 @@ $products = array(
     new \PantsProduct('The Black and Tan Trouser', 50),
     new \PantsProduct('Antarctic Snow Pants (in leopard seal print)', 99),
     new \PantsProduct('South Shore Swim Shorts', 49),
-    new \PantsProduct('Starfish Halloween Costume', 35)
+    //new \PantsProduct('Starfish Halloween Costume', 35)
 );
 
 try {
